client/src/app/components/managing-body/managing-body.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'op-managing-body',
+  templateUrl: 'app/components/managing-body/managing-body.component.html',
+  styleUrls: ['app/components/managing-body/managing-body.component.scss'],
```

Comment:
@tarieli1 I want to merge this PR, so let's don't wait. Remove the css path, and delete the SCSS file for now (because we don't use it). We'll create a new issue to work on the style of the front page.